helm/cluster-api-provider-cloud-director/templates/networking.k8s.io_v1_networkpolicy_capvcd-controller-manager.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.giantswarm.io/branch: '{{ .Values.project.branch }}'
+    app.giantswarm.io/commit: '{{ .Values.project.commit }}'
+    app.kubernetes.io/instance: '{{ .Release.Name }}'
+    app.kubernetes.io/managed-by: '{{ .Release.Service }}'
+    app.kubernetes.io/name: '{{ .Chart.Name }}'
+    app.kubernetes.io/version: '{{ .Chart.AppVersion }}'
+    cluster.x-k8s.io/provider: infrastructure-vcd
+    control-plane: controller-manager
+    helm.sh/chart: '{{ .Chart.Name }}'
+  name: capvcd-controller-manager
+  namespace: '{{ .Release.Namespace }}'
+spec:
+  egress:
+  - ports:
+    - port: 53
+      protocol: UDP
+    - port: 1053
+      protocol: UDP
+    - port: 53
+      protocol: TCP
+    - port: 1053
+      protocol: TCP
+    to:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: kube-system
+      podSelector:
+        matchLabels:
+          app.kubernetes.io/name: coredns
```

Comment:
kubelet and image pulling from various registries should not be controlled by cillium network policies, right? If we ever gonna use the virtual kubelet (running as pod), then we should start care.

Reply:
Image pulling is done by containerd I think which is runnign outside kubernetes. Same goes for kubelet - it's running outside kubernetes.